<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewsItemWidgetClass</class>
 <widget class="QWidget" name="NewsItemWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>75</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>NewsItemWidget</string>
  </property>
  <widget class="QLabel" name="lbl_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>741</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>新闻标题</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_date">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>50</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2024/06/22</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_author">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>瞎编的</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
